### DISCUSSION TITLE
TreeTable Row reordering

### DISCUSSION THREAD

**Discussion by 1Map** [primefaces/community#2744] [PrimeVue]
I use most of the components but had to get rid of the TreeTable due to unable to reorder rows.  Although the documentation state that, I tried everything but could not get my TreeTable to reorder rows (dragging).  It should also have options like drag allow, drop allow, etc.  where you can only drag a child within a group or outside a group.  Also drag a group, again inside the parent only or outside, etc.

https://primevue.org/treetable/#api.column.props.rowReorder

**Comment by nguyentiengiang** [comment]
I have same problems too ~~

**Comment by ericwr** [comment]
Hello, nothing new here? I have the same need.

**Comment by cagataycivici** [comment]
That property is utilized by DataTable, not TreeTable unfortunately.

**Reply by 1Map** [reply]
@cagataycivici Thanks, then why does your documentation have a property like under TreeTable?

See this link:

https://primevue.org/treetable/#api.column.props.rowReorder

**Reply by ericwr** [reply]
Thanks for the reply, like @1Map I thought it was available from the documentation
In any case, it would be a major improvement :)

**Reply by dev-mohieb** [reply]
Is it complex / hard to add for TreeTable? I need that functionality as well.

DataTable doesn't support the same tree structure, so I can't use it for this.
